fdbserver/storageserver.actor.cpp

```diff
@@ -5639,14 +5659,17 @@ void setAssignedStatus(StorageServer* self, KeyRangeRef keys, bool nowAssigned)
 }
 
 void StorageServerDisk::clearRange(KeyRangeRef keys) {
+	kvThroughput->add(keys.expectedSize());
```

Comment:
I would measure the bytes counter for each type of operations, e.g., clearRange, writeKeyValue.